code/game/turfs/turf.dm

```diff
@@ -202,6 +202,32 @@ GLOBAL_LIST_EMPTY(station_turfs)
 			return TRUE
 	return FALSE
 
+/**
+ * Checks whether the specified turf is blocked by something dense inside it, but ignores anything with the climbable trait
+ *
+ * Works the same way is_blocked_turf() works, except with an extra check for TRAIT_CLIMBABLE in atoms. So lockers
+ * and walls will return blocked, but not tables.
+ */
+/turf/proc/is_blocked_turf_ignore_climbable(exclude_mobs = FALSE, source_atom = null, list/ignore_atoms)
```

Comment:
Are the parameters necessary? They don't seem ever used.

Reply:
I figured I might as well leave it open in case anyone else wants to use this version. I can remove them if you'd prefer.

Reply:
If it's unused, remove it.
If someone needs to expand it, let them do it.
We have too much unused bloat already.